streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -965,34 +985,14 @@ private void removeSuspendedTasks() {
             // Close task and state manager
             for (final AbstractTask task : suspendedTasks.values()) {
                 task.close();
-                task.flushState();
```

Comment:
because it is already done in `suspendTasksAndState` so there is no need to flush again.